phys2denoise/metrics/chest_belt.py

```diff
@@ -1,33 +1,38 @@
 """Denoising metrics for chest belt recordings."""
 import numpy as np
 import pandas as pd
+from physutils import io, physio
 from scipy.interpolate import interp1d
 from scipy.stats import zscore
 
 from .. import references
 from ..due import due
 from .responses import rrf
 from .utils import apply_function_in_sliding_window as afsw
-from .utils import convolve_and_rescale, rms_envelope_1d
+from .utils import convolve_and_rescale, return_physio_or_metric, rms_envelope_1d
 
 
 @due.dcite(references.BIRN_2006)
-def respiratory_variance_time(resp, peaks, troughs, samplerate, lags=(0, 4, 8, 12)):
+@return_physio_or_metric()
+@physio.make_operation()
+def respiratory_variance_time(
+    data, fs=None, peaks=None, troughs=None, lags=(0, 4, 8, 12), **kwargs
+):
     """
     Implement the Respiratory Variance over Time (Birn et al. 2006).
 
     Procedural choices influenced by RetroTS
 
     Parameters
     ----------
-    resp: array_like
-        respiratory belt data - samples x 1
-    peaks: array_like
-        peaks found by peakdet algorithm
-    troughs: array_like
-        troughs found by peakdet algorithm
-    samplerate: float
-        sample rate in hz of respiratory belt
+    data : Physio_like
```

Comment:
Could we change this to `physutils.Physio, np.ndarray, or array-like object` instead?
That's also possible now, right?

Reply:
yeap, done